volume/devicemapper/devicemapper.go

```diff
@@ -677,6 +716,14 @@ func (d *DeviceMapperDriver) ensureInitialized() error {
 	return nil
 }
 
+func (d *DeviceMapperDriver) deleteDevice(deviceName string, syncDelete bool) error {
+	if deviceName == "base" || deviceName == "" {
```

Comment:
If we fail to delete the device if the hash is empty, should we clean up the device data anyway? Or somehow label it as a defunct volume?